docs/source/schemas/index.rst

```diff
@@ -169,6 +169,16 @@ characteristics, among other things. The supported types are:
   to each other won't be written next to each other.  This can be good because
   it will spread out the load of writes to many different ranges.
 
+.. [4]
+  If a type does not support range-scanning, **range scans will still be available
+  in the API, but will not behave as expected.** For example, if you have a key with
```

Comment:
I would rephrase this:

"For VAR_STRING and SIZED_BLOB, range-scanning will only exact match components. For example, if you have ... and search for prefix ab: "